### ISSUE TITLE
NFS provisioner installation issue

### ISSUE THREAD

**Issue by GP-1994** [kubernetes-sigs/nfs-subdir-external-provisioner#103]
Hi Team,

While installing NFS provisioner, the status is shown as Deployed as shown below. But I couldn't see any pods running under the namespace. Kindly help to fix this issue.

**Status**:
Name : nfs-subdir-external-provisioner
Namespace : default
Status: deployed
Revision: 1

**Issue**:
No resource found in default namespace

**Comment by yonatankahana** [comment]
can you show us the output of `kubectl describe all -Al chart=nfs-subdir-external-provisioner`?

**Comment by GP-1994** [comment]
> can you show us the output of `kubectl describe all -Al chart=nfs-subdir-external-provisioner`?

While running this, the output was "No resource found".

**Comment by yonatankahana** [comment]
yes, my bad: `kubectl describe all -Al app=nfs-subdir-external-provisioner`
